src/Middleware/Localization/src/RequestCultureProvider.cs

```diff
@@ -16,14 +16,14 @@ public abstract class RequestCultureProvider : IRequestCultureProvider
         /// Result that indicates that this instance of <see cref="RequestCultureProvider" /> could not determine the
         /// request culture.
         /// </summary>
-        protected static readonly Task<ProviderCultureResult> NullProviderCultureResult = Task.FromResult(default(ProviderCultureResult));
+        protected static readonly Task<ProviderCultureResult?> NullProviderCultureResult = Task.FromResult(default(ProviderCultureResult));
```

Comment:
💡 I've seen this a couple times. If there is a shared library where a `SpecializedTasks` class could live, consider something like this:

https://github.com/dotnet/roslyn/blob/4e0ac646bfa6acf43ee165f7c46d4d3a46c0e719/src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/SpecializedTasks.cs#L31-L32

Reply:
Thanks for the suggestion. It's not a lot of code, so I'll keep this until someone complains.